src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -49,15 +66,15 @@ export default class Database extends Emittery {
   }
 
   initialize = async () => {
-    const levelupOptions: any = {
-      keyEncoding: "binary",
-      valueEncoding: "binary"
-    };
     const store = this.#options.db;
-    let db: LevelUp;
+    let db: GanacheLevel;
     if (store) {
-      this.#rootStore = encode(store as AbstractLevelDOWN, levelupOptions);
-      db = levelup(this.#rootStore, {});
+      if (typeof store === "string") {
+        db = <GanacheLevel>new Level(store, LEVEL_OPTIONS);
```

Comment:
```suggestion
        db = new Level(store, LEVEL_OPTIONS) as GanacheLevel;
```
For consistencies sake